.github/workflows/release-workflow.yml

```diff
@@ -0,0 +1,115 @@
+# ---------------------------------------------------------------------------
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: release
+
+on:
+  workflow_call:
+    inputs:
+      ref:
+        required: true
+        type: string
+      goVersion:
+        required: true
+        type: string
+      javaVersion:
+        required: true
+        type: string
+    secrets:
+      registryUsername:
+        required: true
+      registryPassword:
+        required: true
+
+jobs:
+  release:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Set up JDK ${{ inputs.javaVersion }}
+        uses: actions/setup-java@v2
+        with:
+          version: ${{ inputs.javaVersion }}
+          distribution: "temurin"
+      - name: Install Go ${{ inputs.goVersion }}
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ inputs.goVersion }}
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ inputs.ref }}
+      - name: Cache modules
+        uses: actions/cache@v1
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      - name: Get nightly version
+        shell: bash
+        run: |
+          V=$(make get-version | sed s/-SNAPSHOT/-nightly/)
+          echo "VERSION=$V" >> $GITHUB_ENV
+      - name: Global Env
+        run: |
+          echo "Using VERSION=${{ env.VERSION }}"
+
+          IMAGE_NAME=docker.io/${TEST_DOCKER_HUB_USERNAME:-testcamelk}/camel-k
+          echo "Using IMAGE_NAME=$IMAGE_NAME"
+          echo "IMAGE_NAME=$IMAGE_NAME" >> $GITHUB_ENV
+
+          MAVEN_REPOSITORY=$(make get-staging-repo)
+          echo "Using MAVEN_REPOSITORY=$MAVEN_REPOSITORY"
+          echo "MAVEN_REPOSITORY=$MAVEN_REPOSITORY" >> $GITHUB_ENV
+
+      - name: Login to Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
```

Comment:
I guess it should login to docker hub instead

Reply:
fixed